lessons/Early-Math/associative-property/langs/en-us/pages/discover-assoc-cont.adoc

```diff
@@ -0,0 +1,62 @@
+= Discover the Associative Property (2)
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Evaluate the two Circles of Evaluation below to help you decided if the Associative Property holds for problems involving *multiplication*:
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(* (* 2 5) 10))} | @show{(coe  '(* 2 (* 5 10)))}
```

Comment:
Again I think that this would be even stronger with  3 examples.

And also perhaps making one of the numbers a little more complex to work with so that we see the advantage of regrouping

eg

(2 x 7) x 5 ... 2 x ( 7 x 5)   ... (2 x 5) x 7